### PR TITLE
refactor database config connection and backend restructure

### PR DESCRIPTION
**Pull request by jfv-2000**
-Removed existing database connection configuration to a new structured one
-Added new folders to restructure the backend
-Model folder for sequelize 
-Views folder added for jade view engine (simple back-end visualization)
-New route added in /routes/users as example on how to get data from database

-implementation of Login and Register in the back-end using JWT for token validity
-connection of Front-end login requests to back-end done

### CODE REVIEW

File: backend/config/database.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const Sequelize = require('sequelize');
+
+//config to link sequelize with the database
+module.exports = new Sequelize('MinicapDatabase', 'minicap', 'soen390!', {
```
Comment (NicholasYip) [inline backend/config/database.js:4]: Really glad we're using Sequelize!

**Comment by NicholasYip** [comment]
tested locally, LGTM!

**Review by NicholasYip — APPROVED** [review]

**Review by SergioAAI — APPROVED** [review]

**Comment by SergioAAI** [comment]
Looks good to me

**Comment by stevenvuong00** [comment]
looks really good to me, love the work, keep up the progress

**Review by stevenvuong00 — APPROVED** [review]

**Comment by KelvinTrang** [comment]
Very good. Good job

**Review by KelvinTrang — APPROVED** [review]

**Pull request merged**